src/twisted/web/newsfragments/12233.bugfix

```diff
@@ -1 +1 @@
-The HTTP 1.0/1.1 server provided by twisted.web is now more picky about the first line of a request, improving compliance with RFC 9112.
+twisted.web's HTTP/1.1 server is now more picky about the first line of a request, improving compliance with RFC 9112.
```

Comment:
I think that we should revert the changes to 12233 and 12235.

If these changes are needed, they should be done on the NEWS file inside the release pr https://github.com/twisted/twisted/pull/12272

Reply:
Absolutely, I'll take care of that when I merge forward.